RELEASES.md

```diff
@@ -1,3 +1,109 @@
+Version 1.38.0 (2019-09-26)
+==========================
+
+Language
+--------
+- [The `#[global_allocator]` attribute can now be used in submodules.][62735]
+- [The `#[deprecated]` attribute can now be used on macros.][62042]
+
+Compiler
+--------
+- [Added support for the `armv7-unknown-linux-gnueabi` and
+  `armv7-unknown-linux-musleabi` targets.][63107]
+- [Added support for the `hexagon-unknown-linux-musl` target.][62814]
+- [Added pipelined compilation support to `rustc`.][62766] This will
+  improve compiliation times in some cases. For further information please refer
+  to the [_"Evaluating pipelined rustc compilation"_][pipeline-internals] thread.
+- [Added support for the `riscv32i-unknown-none-elf` target.][62784]
+
+
+Libraries
+---------
+- [`ascii::EscapeDefault` now implements `Clone` and `Display`.][63421]
+- [Derive macros for prelude traits (e.g. `Clone`, `Debug`, `Hash`) are now
+  available at the same path as the trait.][63056] e.g. The `Clone` derive macro
+  is available at `std::clone::Clone`.
+- [`str::Chars` now implements `Debug`.][63000]
+- [`slice::{concat, connect, join}` now accepts `&[T]` in addition to `&T`.][62528]
+- [`*const T` and `*mut T` now implement `marker::Unpin`.][62583]
+- [`Arc<[T]>` and `Rc<[T]>` now implement `FromIterator<T>`.][61953]
+- [Built-in macros (e.g. `include!`, `line!`) are now available through the
+  `std` and `core` prelude.][62086] e.g. `core::prelude::v1::include`.
```

Comment:
I'd omit this item.
It's more important to mention that built-in macros are now available through std and core root like `core::include!(...)`, which is a part of the same PR as "Derive macros for prelude traits..." above (so I'd put it closely to that item).

Reply:
Okay, I've removed this item and reworded the earlier PR to include what you mentioned.